api/rpc/account/account.proto

```diff
@@ -146,7 +146,8 @@ message SignUpRequest {
 
 message VerificationRequest {
   string name = 1;
-  string code = 2;
+  string password = 2;
```

Comment:
Discusion: 
http://security.stackexchange.com/questions/53594/why-is-client-side-hashing-of-a-password-so-uncommon

If the password hash is sent in place of the password then the password hash becomes in effect a plain text password stored in the database.  Which means if the database if compromised then the accounts can be accessed.

It's also possible to double hash (client side and server side) but that only helps if the user reuses the password and either the tls or the server is compromised.